sw/device/tests/sim_dv/entropy_src_fuse_en_fw_read_test.c

```diff
@@ -61,6 +61,9 @@ static void entropy_with_fw_override_enable(dif_entropy_src_t *entropy_src) {
       .fips_enable = true,
       .route_to_firmware = true,
       .single_bit_mode = kDifEntropySrcSingleBitModeDisabled,
+      .health_test_threshold_scope = false, /*default*/
```

Comment:
Inline comments from the style guide should be in this format: `// default`.  I'd also more details to these comments to describe in more detail why we are doing this, or otherwise remove the comment all together.